tests/atlas.phpt

```diff
@@ -21,28 +71,41 @@ $envs = [
 	'ATLAS_TLS12',
 	'ATLAS_SRV_TLS12',
 ];
-
-$command = new \MongoDB\Driver\Command(['ping' => 1]);
-$query = new \MongoDB\Driver\Query([]);
+$x509Envs = [
+	'ATLAS_X509',
+	'ATLAS_X509_DEV',
+];
 
 foreach ($envs as $env) {
 	echo $env, ': ';
-	$uri = getenv($env);
+	$uri = extractUri($env);
```

Comment:
It looks like the only purpose of `extractUri()` is to convert `false` return value from `getenv()` into `null`. That seems unnecessary given the conditional below that fails a non-string value.

Reply:
Good point, I was a little eager with my refactoring. Removed it in favour of `getenv`.